addon/src/lib/appStartup.js

```diff
@@ -1,30 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
bootstrap-removal, `sessionstore-windows-restored` was a hack to speed up startup for hybrid bootstrap embedded webextensions.